test/Microsoft.AspNet.Mvc.FunctionalTests/Compiler/Resources/MvcTagHelpersWebSite.MvcTagHelper_Home.Order.html

```diff
@@ -18,6 +18,7 @@
         </div>
         <div>
             <label class="order" for="Products">Products</label>
+
```

Comment:
added a c# statement line to the corresponding .cshtml file (last file on this page)